framework/src/st_things/things_stack/things_stack.c

```diff
@@ -317,6 +317,8 @@ bool try_turn_on_soft_ap()
 	wifi_manager_info_s info;
 	wifi_manager_get_info(&info);
 
+	wifi_manager_remove_config();
```

Comment:
When this function gets failed, what happens?

Reply:
There is no big problem even if wifi_profile information is not erased.
st_things maintains provisioning data separately and requests join to sta home ap through the information.
Also, if Easy-Setup has been resumed after the reset, it will not be a problem to save the profile again.